editor/components/inserter/no-blocks.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+function NoBlocks() {
+	return (
+		<span className="editor-inserter__no-results">
```

Comment:
I think the classname should be `editor-inserter__no-blocks` to match our guidelines

Reply:
oops my bad